tests/integ/modin/frame/test_take.py

```diff
@@ -12,7 +12,10 @@
 
 
 @pytest.mark.parametrize("test_multiindex", [False, True])
-def test_df_take(float_native_df, test_multiindex):
+@pytest.mark.parametrize("dtype", ["float", "timedelta64[ns]"])
```

Comment:
maybe just have two different dataframes that we can use? Minor nit, feel free to ignore.